proto/spire/api/server/local_authority/v1/local_authority.proto

```diff
@@ -0,0 +1,174 @@
+syntax = "proto3";
+package spire.api.server.local_authority.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/server/local_authority/v1;local_authorityv1";
+
+// The LocalAuthority service provides a way to manage the signing keys (and
+// related material) of the SPIRE Server exposing it.
+service LocalAuthority {
+        // GetJWTAuthorityState returns the state of all locally configured
+        // JWT authorities.
+        rpc GetJWTAuthorityState(GetJWTAuthorityStateRequest) returns (GetJWTAuthorityStateResponse);
+
+        // PrepareJWTAuthority prepares a new JWT authority for use by
+        // generating a new key and injecting it into the bundle. This action
+        // will propagate the new public key cluster-wide.
+        rpc PrepareJWTAuthority(PrepareJWTAuthorityRequest) returns (PrepareJWTAuthorityResponse);
+
+        // ActivateJWTAuthority activates a prepared JWT authority for use,
+        // which will cause it to be used for all JWT signing operations
+        // serviced by this server going forward. If a new JWT authority has
+        // not already been prepared, a FailedPrecondition error will be returned.
+        rpc ActivateJWTAuthority(ActivateJWTAuthorityRequest) returns (ActivateJWTAuthorityResponse);
+
+        // TaintJWTAuthority marks the previously active JWT authority as
+        // being tainted. SPIRE Agents observing an authority to be tainted
+        // will perform proactive rotations of any key material related to
+        // the tainted authority. The result of this action will be observed
+        // cluster-wide.
+        //
+        // If a previously active JWT authority does not exist (e.g. if one
```

Comment:
I'm not sure we can be in the state, at least with the way the manager is organized. It will automatically activate a prepared authority if there is no active authority....